sonar/theme/assets/scss/common/_theme.scss

```diff
@@ -1,10 +1,12 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap');
```

Comment:
I think we should host the font files too, for privacy concerns.

Reply:
you're right, thanks